### PR TITLE
updates syntax and replaces spotify with official node-spotify-api

### PR DESCRIPTION
**Pull request by lhocke**
spotify will require manually putting in keys to make sure it connects but should be more stable than previous package.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "express": "4.13.4",
     "mongojs": "2.4.0",
     "morgan": "1.7.0",
+    "node-spotify-api": "^1.0.7",
     "request": "2.81.0",
```
Comment (pavankat) [inline package.json]: can you get rid of the spotify package?

File: client/src/components/Movie.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ class Movie extends Component {
 
   render() {
   	let moviePoster = '';
-  	if (this.state.movie.Poster != 'N/A') moviePoster = <img src={this.state.movie.Poster} />;
```
Comment (pavankat) [inline client/src/components/Movie.js]: can you add the title of the movie in the alt

File: server.js [hunk 3]
```diff
@@ -3,9 +3,13 @@ var express = require("express");
 var mongojs = require("mongojs");
 var logger = require("morgan");
 var bodyParser = require('body-parser');
-var spotify = require("spotify");
+var Spotify = require("node-spotify-api");
 var request = require('request');
 var path = require('path');
```
Comment (pavankat) [inline server.js]: can you use environment variables here

**Comment by lhocke** [comment]
environment variables added

**Comment by pavankat** [comment]
thanks man!

**Pull request merged**